services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
```

Comment:
It accepts Optional[list[int]] because None as default parameter
Fix in all places(with all types)